### ISSUE TITLE
Test failure with hashable 1.3.1.0

### ISSUE THREAD

**Issue by felixonmars** [snapframework/snap-core#303]
It passes with hashable 1.3.0.0 and others unchanged.

```
Snap.Util.CORS.Tests:
  CORS/simple: [OK]
  CORS/options: [Failed]
Header "Access-Control-Allow-Headers"
expected: Just "X-STUFF, Content-Type"
 but got: Just "Content-Type, X-STUFF"
```

**Comment by kozross** [comment]
This is due to the hash being unstable. I've sent a PR repairing this test, but in general, we shouldn't rely on any order we don't ourselves enforce.

[Reference about hash instability](https://hackage.haskell.org/package/hashable-1.3.2.0/docs/Data-Hashable.html#t:Hashable).

**Comment by ysangkok** [comment]
I think this can be closed since 94af153cc473015a5af321b7ba5d8de65580290c

**Comment by felixonmars** [comment]
Confirmed that the new version work fine. Thanks!
